### PR TITLE
Update mkl pin to 2019

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR updates the `mkl` pin to `2019`.

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
I'm not really sure what to do re: migration, I presume this probably does need one.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,11 +9,6 @@ Checklist
 * [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
 * [ ] Ensured the license file is being packaged.
 
-### Migration
-<!--
-If this is an ABI incompatible change which requries a migration, please note it and ping ``@conda-forge/bot``.
--->
-
```
Comment (isuruf) [inline .github/PULL_REQUEST_TEMPLATE.md]: Can you revert this change?

Comment (duncanmmacleod) [reply .github/PULL_REQUEST_TEMPLATE.md]: @isuruf, that change was made by `conda smithy`, so I've just removed that entire commit.

**Comment by CJ-Wright** [comment]
@duncanmmacleod would you be able to follow the steps [here](https://regro.github.io/cf-scripts/migrators.html#adding-to-auto-tick-xsh) to make a migrator for MKL?

**Comment by isuruf** [comment]
There's only 2 feedstocks needing this. `gmsh` and `kwant`. Both are using only standard BLAS, LAPACK API which means a blas migrator would be sufficient

**Comment by isuruf** [comment]
There's only 2 feedstocks needing this. `gmsh` and `kwant`. Both are using only standard BLAS, LAPACK API which means a blas migrator would be sufficient

**Comment by CJ-Wright** [comment]
Where is blas moving to?

**Comment by isuruf** [comment]
It's a more complex migration. https://github.com/conda-forge/conda-forge.github.io/pull/734
